src/main/java/com/exadel/discount/dto/coupon/CouponDto.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.discount.dto.coupon;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CouponDto {
+    @NotNull
+    private UUID id;
+    @NotNull
+    private LocalDateTime date;
```

Comment:
I think you can add full DiscountDTO or discount id, name and promo for example, user user will be able to see this information

Reply:
Done